File: Problem1/prob1.py

#number = 1000
#numArray = []
#answer = 0

#for i in range(number):
#    if i % 3 == 0:
#        numArray.append(i)

#    elif i % 5 == 0:
#        numArray.append(i)


#print(numArray)
#for num in numArray:
#    answer += num

answer = sum(i for i in range(1000) if i%3 == 0 or i % 5 == 0)

print(answer)

File: Problem1/prob1.cpp
#include <iostream>
#include <vector>
#include <numeric>


int main(){

    int number = 1000, i = 0, answer = 0;
    std::vector<int> arr;

    while(i < number){

        if ((i % 3 == 0) || (i % 5 ==0)){
            arr.push_back(i);}
            
        i++;
    }

    answer = std::accumulate(arr.begin(), arr.end(), 0);
    std::cout<<answer<<std::endl;



}
